### DISCUSSION TITLE
Issuetemplate for product specific issues

### DISCUSSION THREAD

**Discussion by stephanbcbauer** [eclipse-tractusx/sig-release#230] [Ideas]
There are currently already many and good feature-level templates in this repository.

Is there also a template for the product-specific issues somewhere?

**Comment by FaGru3n** [comment]
Hi @stephanbcbauer, currently there is no product related template prepared within sig-release.

This reduce a lot the maintaining effort to maintain the templates for each product and as we already have the product labels for each product to define for which component/product the issue is raised to. Also with a bunch of Templates we would also confuse a lot of people.

The current idea ensures that you don´t have to know each product exactly for requesting a Feature or Bug. So everyone can request something, no matter which and how many products are involved or you have to fullfill each involed product template.

I would assume that the PM / PO's need a default Markdown Template, what to fill out for a Feature i would guide them to personal saved `reply's`
https://docs.github.com/en/get-started/writing-on-github/working-with-saved-replies/about-saved-replies

[Create a saved reply](https://docs.github.com/en/get-started/writing-on-github/working-with-saved-replies/creating-a-saved-reply)

If the general templates Feature or Bug is not sufficient or should be modifed then everyone can create a PR against it or ask for changes.

**Comment by stephanbcbauer** [comment]
answered. THX
